test/e2e/create_test.go

```diff
@@ -176,6 +196,24 @@ func (tc *testContext) testBYOHConfiguration(t *testing.T) {
 		err := tc.waitForWindowsNodes(gc.numberOfBYOHNodes, false, false, true)
 		assert.NoError(t, err, "Windows node creation failed")
 	})
+	// Make a best effort attempt to collect logs from each BYOH instance
+	tc.byohLogCollection()
+}
+
+// byohLogCollection kicks off the collection of logs for instances listed in the windows-instances configmap
+func (tc *testContext) byohLogCollection() {
+	windowsInstances, err := tc.client.K8s.CoreV1().ConfigMaps(tc.namespace).Get(context.TODO(),
+		wiparser.InstanceConfigMap, metav1.GetOptions{})
+	if err != nil {
+		log.Printf("unable to collect logs, error retrieving instances ConfigMap: %v", err)
+		return
+	}
+	for addr := range windowsInstances.Data {
+		err = tc.nodelessLogCollection("byoh", addr)
```

Comment:
Should we append the address to the job name instead of just `print-logs-byoh`?

Reply:
+1